src/extension.ts

```diff
@@ -91,4 +92,48 @@ export async function activate(context: vscode.ExtensionContext): Promise<Extens
     };
 }
 
+function createQuickPickMenu(credentialController: CredentialController, buildController: BuildController) {
+    const quickPickMenu = vscode.window.createQuickPick();
+    if (credentialController.credential.signInStatus === 'SignedOut') {
+        quickPickMenu.items = <vscode.QuickPickItem[]>[
+            {
+                label: 'Sign-in',
+                description: 'Sign-in to Docs build system',
+                picked: true
+            },
+        ];
+    } else {
```

Comment:
>else [](start = 6, length = 4)

else (status === 'SingedIn')? I remember we have some other intermediate status. To avoid corner case, we should use exactly match status